src/pages/ChatFinderPage/index.tsx

```diff
@@ -93,10 +93,10 @@ function ChatFinderPage({betas, isSearchingForReports, navigation}: ChatFinderPa
                 headerMessage: '',
             };
         }
-        const optionList = OptionsListUtils.getSearchOptions(options, '', betas ?? []);
+        const optionList = OptionsListUtils.getSearchOptions(options, debouncedSearchValueInServer.trim(), betas ?? []);
```

Comment:
Why this change?

Reply:
We weren't passing the search value into search options for whatever reason and it was preventing `isSearchingForReports` from being `true` since that depends on the search value.

I checked over the logic of how it's used and it shouldn't affect our results in the chat finder page.

Reply:
Not sure I have full grasp on the current logic, but here's what I'm seeing:

- We get the options list without the search value
- We then filter those options in a separate step below

I'm not sure why we do it that way exactly. But it seems possibly intentional that we are passing an empty string here in the spot you are modifying.

Reply:
also we got rid of `isSearchingForReports` so maybe this is not needed for that reason.

Reply:
Got rid of this!